README.md

```diff
@@ -0,0 +1,94 @@
+![inspectIT OpenCensus Edition](https://inspectit.github.io/inspectit-oce/images/inspectit-oce.png)
+
+# inspectIT OpenCensus Edition
+
+[![Build Status](https://travis-ci.com/inspectIT/inspectit-oce.svg?branch=master)](https://travis-ci.com/inspectIT/inspectit-oce)
+[![Code Coverage](https://codecov.io/gh/inspectit/inspectit-oce/branch/master/graph/badge.svg)](https://codecov.io/gh/inspectIT/inspectit-oce)
+
+inspectIT OCE is a zero-configuration Java agent for dynamically collecting application performance, tracing and behavior data based on the [OpenCensus library](https://opencensus.io/).
+In order to use inspectIT OCE, the source code of the target application does **not** have to be touched, changed or modified - even access to the actual source code is not required.
+It automatically injects all required components and executes the necessary actions by itself.
+
+But wait - isn't there already an inspecIT existing?
+Yes, it is! Please [read this](http://www.inspectit.rocks/public-announcement-inspectit-future-plans-and-road-map/) in order to understand why we are developing a new and improved version of it!
+Compared to the former [inspectIT](https://inspectit.rocks/), the OC edition follows the approach of focusing on compatibility and interaction with other well-known and established open source tools like [Elasticsearch](https://www.elastic.co/products/elasticsearch), [Prometheus](https://prometheus.io/) or [InfluxDB](https://www.influxdata.com/).
+This reduces the amount of components which have to be introduced into an existing infrastructure or which needs to be familiarized with.
+
+## Collected Data
+
+The inspectIT OCE Java agent collects a lot of different data. Currently among others the following data is collected:
+
+* Metrics
+  * CPU (usage and number of cores)
+  * Disk Space (used, free and total)
+  * Memory (used and available for various regions like heap or non-heap)
+  * Threads (counts and states)
+  * Garbage Collection (pause times and collection statistics)
+  * Class Loading (loaded and unloaded counts)
+  
+ A full list of gathered data and metrics can be found in the [documentation](http://docs.inspectit.rocks/releases/latest/#_metrics).
+
+## Installation

```

Comment:
Before installation, I would add a quick start that explains how to try out the demo. The demo should be the first choice, before really checking it with your app